pkg/tsdb/azuremonitor/loganalytics/azure-log-analytics-datasource.go

```diff
@@ -107,21 +108,26 @@ func (e *AzureLogAnalyticsDatasource) buildQueries(logger log.Logger, queries []
 
 		apiURL := getApiURL(queryJSONModel)
 
-		params := url.Values{}
 		rawQuery, err := macros.KqlInterpolate(logger, query, dsInfo, azureLogAnalyticsTarget.Query, "TimeGenerated")
 		if err != nil {
 			return nil, err
 		}
-		params.Add("query", rawQuery)
 
+		resources := []string{}
+		if azureLogAnalyticsTarget.Resource != "" {
+			resources = []string{azureLogAnalyticsTarget.Resource}
+		}
+		if len(azureLogAnalyticsTarget.Resources) > 0 {
```

Comment:
nit: I would swap these two conditions around and use an `else`. If the `Resources` property has a non-zero length then we shouldn't need to check the value of the `Resource` prop (unless I'm misunderstanding?).